invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html

```diff
@@ -24,10 +24,11 @@
           </div>
           <h3 class="section-title">{{ _("Configure") }}</h3>
           <div class="command-line">
-            {% trans configuration_link="https://inveniordm.docs.cern.ch/install/configuration/" %}Configure the
-            datamodel, resource types, permissions and much more to make it suit your needs! For more
-            information on how to do it visit the <a href="{{ configuration_link }}">configuration
-              section in the docs</a>.{% endtrans %}
+            {% trans configuration_link="https://inveniordm.docs.cern.ch/install/configuration/" %}
```

Comment:
Hmm I think we need to re-write this text. Because what it mentions (data model, resource types, premissions, etc.) have been moved to the [customize](https://inveniordm.docs.cern.ch/customize/) section. The [Configure](https://inveniordm.docs.cern.ch/install/configuration/) is quite minimalistic.

Proposal:

```
<a href="{{ configuration_link }}" target="_blank">Configure</a> your instance's theme templates, services' locations and credentials and more.
```
 
Very nice on the `_blank` 🚀